### PR TITLE
Magiclysm - Introducing Channel Wands and Disposable Wands

### PR DESCRIPTION
**Pull request by foulman**
Introduces Channel Wands and Disposable Wands.

#### Summary
```SUMMARY: Mods "Magiclysm - Introduces Channel and Disposable wands"```

#### Purpose of change
Triple the number of wands available, with different means of powering them. Also, wands can be shoved into your belt now and don't conduct electricity. No-brainers.

#### Describe the solution
Channel wands utilize the USE_PLAYER_ENERGY flag, so any spells inside them are powered by the wielder.
Disposable wands are just what it says, they don't hold mana crystals, they hold a certain number of charges, and when they're expended, the wand is empty and useless.

#### Describe alternatives you've considered
Not making more items.

#### Testing
I spawned and tested wands of each kind, there's nothing complicated here.

#### Additional context
2/5

### CODE REVIEW

File: data/mods/Magiclysm/enchanted/enchanted_wands.json [hunk 1]
```diff
@@ -189,5 +222,353 @@
...
     "use_action": { "type": "cast_spell", "spell_id": "cone_cold", "no_fail": true, "level": 15, "need_wielding": true }
+  },
+  {
+    "type": "TOOL",
+    "id": "wand_chan_magic_missile_minor",
+    "name": "channeling minor wand of magic missile",
+    "name_plural": "channeling minor wands of magic missile",
+    "copy-from": "chan_wand",
+    "description": "A slender wooden wand with a mana channel at the base that casts a spell when activated.  This wand casts magic missile.",
+    "use_action": { "type": "cast_spell", "spell_id": "magic_missile", "no_fail": true, "level": 5, "need_wielding": true }
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_wands.json]: if you're channeling it, why does it reduce the failure chance to 0? this makes these items much better than casting the spell at all.

Comment (foulman) [reply data/mods/Magiclysm/enchanted/enchanted_wands.json]: They are, yes. Basically what I did was create 3 tiers of wands, disposable, rechargeable, and channeling.
Channeling is at the top, it's just a wand that is designed and created solely to take mana you pump into it and put a spell out the other end, the downside is you're paying for it, which can be costly for say cone of cold.. the upside being it won't fail. What is being channeled is the mana/cost/power.
You are still holding a wand though and you can't cast other spells while doing so.
I try to think of magic items more like electronics, it's a magical circuit that performs the function it was designed to do. In this case, shoot magic missiles.
As far as it being much better than casting the spell yourself, this is why we created spears, bows, slings, and guns. Tools do a better job than just using your hands.

Comment (KorGgenT) [reply data/mods/Magiclysm/enchanted/enchanted_wands.json]: hm, i request that you separate out the channeling wands from this PR to be discussed later. This is a pretty severe balance decision to be making, and i didn't see anything wrong with the other wands.

File: data/mods/Magiclysm/enchanted/enchanted_wands.json [hunk 2]
```diff
@@ -11,11 +11,44 @@
...
+    "weight": 230,
+    "volume": 2,
+    "price": 10000,
+    "bashing": 3,
+    "to_hit": 1,
+    "material": [ "wood" ],
+    "symbol": "|",
+    "color": "brown",
+    "flags": [ "BELT_CLIP", "NONCONDUCTIVE", "USE_PLAYER_ENERGY" ]
+  },
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_wands.json]: This should be removed from this pr, and be part of channel wands instead.

**Comment by foulman** [comment]
@KorGgenT we gonna get this done?

**Comment by KorGgenT** [comment]
i thought you said you were going to add itemgroups for these?

**Pull request merged**